aiida/common/pluginloader.py

```diff
@@ -125,6 +125,48 @@ def get_plugin(category, name):
 
     return plugin
 
+def load_plugin_safe(base_class, plugins_module, plugin_type):
+    """
+    It is a copy of load_plugin function to return closely related node class
+    if plugin is not available. We are duplicating load_plugin function to not break
```

Comment:
You can remove this sentence "We are duplicating load_plugin function to not break its default behaviour." (as above, it's not a copy).